test/integration/composite-weighted/composite-weighted.test.ts

```diff
@@ -0,0 +1,55 @@
+import { MeasureReport } from 'fhir/r4';
+import { calculateMeasureReports } from '../../../src/calculation/Calculator';
+import { getJSONFixture } from '../helpers/testHelpers';
+
+const MEASURE_BUNDLE: fhir4.Bundle = getJSONFixture('composite-weighted/composite-weighted-bundle.json');
+const PATIENT_COMP1_NUMER_COMP2_NUMER: fhir4.Bundle = getJSONFixture(
+  'composite-all-or-nothing/patients/patient-comp1-numer-comp2-numer-bundle.json'
+);
+const PATIENT_COMP1_DENOM_COMP2_NUMER: fhir4.Bundle = getJSONFixture(
+  'composite-all-or-nothing/patients/patient-comp1-denom-comp2-numer-bundle.json'
+);
+
+describe('Composite measure weighted scoring', () => {
+  it('should calculate weighted result for components', async () => {
+    const { results } = await calculateMeasureReports(
+      MEASURE_BUNDLE,
+      [PATIENT_COMP1_NUMER_COMP2_NUMER, PATIENT_COMP1_DENOM_COMP2_NUMER],
+      {
+        measurementPeriodStart: '2023-01-01',
+        measurementPeriodEnd: '2023-12-31',
+        reportType: 'summary'
+      }
+    );
+
+    expect(results).toBeDefined();
+    const result = results as MeasureReport;
```

Comment:
You could use the [CompositeMeasureReport](https://github.com/projecttacoma/fqm-execution/blob/e532aacf23f6bb1bd3afded4368a475173bb8a87/src/calculation/CompositeReportBuilder.ts#L12) type instead, which enforcese that there only be 1 group with 2 populations, avoiding the need for some question marks